lib/actions/environment-variables.coffee

```diff
@@ -31,19 +34,31 @@ exports.list =
 	]
 	permission: 'user'
 	action: (params, options, done) ->
-		resin.models.environmentVariables.getAllByApplication options.application, (error, environmentVariables) ->
-			return done(error) if error?
+		async.waterfall([
```

Comment:
I'm making a note to make use of promises in all CLI commands that make use of SDK models functions, and will push another PR.